test/tracing/reactnavigation.test.ts

```diff
@@ -66,24 +66,15 @@ describe('ReactNavigationInstrumentation', () => {
         contexts: expect.objectContaining({
           trace: expect.objectContaining({
             data: {
-              route: {
-                hasBeenSeen: false,
-                key: 'initial_screen',
-                name: 'Initial Screen',
-                params: {},
-              },
-              previousRoute: null,
+              [SEMANTIC_ATTRIBUTE_ROUTE_NAME]: 'Initial Screen',
+              [SEMANTIC_ATTRIBUTE_ROUTE_KEY]: 'initial_screen',
+              [SEMANTIC_ATTRIBUTE_ROUTE_HAS_BEEN_SEEN]: false,
               [SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN]: 'manual',
-              [SEMANTIC_ATTRIBUTE_SENTRY_SOURCE]: 'component',
+              [SEMANTIC_ATTRIBUTE_SENTRY_SOURCE]: 'component', // Check why this was component
```

Comment:
Nit to help filtering it in the future

```suggestion
              [SEMANTIC_ATTRIBUTE_SENTRY_SOURCE]: 'component', // TODO: Check why this was component
```